README.md

```diff
@@ -556,6 +556,8 @@ ANSWERS.addComponent('VerticalResults', {
   maxNumberOfColumns: 1,
   // Optional, whether to display the total number of results, default true
   showResultCount: true,
+  // Optional, custom text for the results count. You can specify the variables resultsCountStart, resultsCountEnd, and resultsCount with the double bracket notation e.g. {{resultsCount}}.
+  resultsCountText: '{{resultsCountStart}} - {{resultsCountEnd}} of {{resultsCount}}',
```

Comment:
it seems weird to me that this is classified as a text rather than a template (as it gets compiled). are we 100% about the name `resultsCountText`?

(others are called template like noResults, etc, but I understand those are actually changing the entire component template)

Reply:
ah i read the Slack thread, it seems "plain string" is what we are going for

Reply:
I originally called this resultsCountTemplateString. No, I'm not 100% sure, but since Tom and Rose have decided we don't want to let people pass in arbitrary html, it's difficult to use the word template somewhere since we then have to add additional language explaining that it's not really a template since you can't add random html.

Reply:
Just updated the PR so this isn't confusing. Tom and I discussed off github that we don't want people to insert html here (which I originally thought was functionality we wanted). Sorry I didn't get a chance to do this earlier I was looking at some issues (which turned out to not be issues) with the 1.4.3 and 1.3.3 cut.